src/DependencyInjection/FOSHttpCacheExtension.php

```diff
@@ -131,6 +134,22 @@ public function load(array $configs, ContainerBuilder $container)
         }
     }
 
+    private function loadCacheable(ContainerBuilder $container, array $config)
+    {
+        $container->setParameter(
+            $this->getAlias().'.cacheable.response.additional_status',
+            $config['response']['additional_status']
```

Comment:
should we do that in the else if the expression is empty? or do we want this parameter to always be defined? or we could make it not a container parameter but set the argument, like we do for expression.